tests/README.md

```diff
@@ -117,3 +117,29 @@ docker run \
         --deploy-testing-infra \
         --path-to-testing-infra-manifests=data/manifests
 ```
+
+
+## Skipping Tests When Cluster Requirements Aren't Met
+KubeVirt provides [tests decorators](https://github.com/kubevirt/kubevirt/blob/main/tests/decorators/decorators.go) 
+to indicate the cluster requirements for certain tests. If your 
+cluster doesn't meet these requirements, you can choose to skip 
+the relevant tests by explicitly setting the `KUBEVIRT_E2E_SKIP`
```

Comment:
Can you also briefly mention `KUBEVIRT_E2E_FOCUS` and `KUBEVIRT_LABEL_FILTER`?

Reply:
Done